### ISSUE TITLE
Minor typos in Chap 12

### ISSUE THREAD

**Issue by dilayercelik** [mspeekenbrink/sdam-book#8]
"<s>Fro</s> _For_ a given link function, [...]"
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L62

"[...] analytical solutions for maximum likelihood parameters of generalized linear models <s>of</s> _are_ usually not available."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L72

"For large-enough n, the difference between the t-distribution and the Normal distribution _is_ so small that [...]"
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L84

"<s>As</s> _A_ straightforward way to compute (approximate) confidence intervals [...]"
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L106

"For non-linear models such as generalized linear models, confidence intervals will often <s>no</s> _not_ be symmetrical [...]"
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L123

"The same can be said for assuming the <s>model</s> model errors (residuals) follow a Normal distribution."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L163

A typo in the following sentence? "On the _log-odds log-odds_ or logits, every one-unit increase in X adds β1 to the log-odds."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L253

"Figure 12.4: The Probit link function equals the standard Normal <s>cumultive</s> _cumulative_ distribution function. <s>THe</s> _The_ solid dark line shows the standard Normal cumulative distribution function, with arrows depicting the .05, .5, .7, and .975 quantiles."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L361-L366

"The main purpose of this additional example is to illustrate that other link functions <s>which</s> can be used."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L414

"Poisson regression is useful for count data where the total number of possible occurrences of an event <s>in</s> _is_ not given."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L451

"For example, suppose you give participants one hour _to_ solve as many algebra problems as they can."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L451

"If the rate is unknown and can, in principle, take any possible value, then <s>the</s> there is no theoretical maximum to the number of algebra problems that can be solved in an hour."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L451

"As we will see later in this Chapter, however, <s>is that</s> the overdispersion is likely due to not properly accounting for the dependencies in the data: [...]"
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L574

Incorrect rendering: "Dealing with this properly requires the use of generalized linear mixed-effects models, which we will introduce in Section **@ref({sec-glmer})**."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L574

Could there be a typo in the following sentence? "When zero-count cells are present, Agresti ([2018](https://mspeekenbrink.github.io/sdam-book/ch-generalized-linear-models.html#ref-agresti2018introduction)) recommends to add a value of 1/2 to all cells of saturated models (including cells with <s>on-zero</s> _non-zero_ counts)."
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L693

Missing parenthesis: "Table [12.11](https://mspeekenbrink.github.io/sdam-book/ch-generalized-linear-models.html#tab:metacognition-glmer) shows the results for a logistic mixed-effects regression model, using data from all participants of the metacognition experiment (see Section [12.5.2]**_)_**(https://mspeekenbrink.github.io/sdam-book/ch-generalized-linear-models.html#sec-logistic-regression-metacognition).
- https://github.com/mspeekenbrink/sdam-book/blob/1e5d012f470a7fd915c601ddbf2be60f262723c6/12-GenLM.Rmd#L796

**Comment by mspeekenbrink** [comment]
Thanks yet again, so helpful!
